app/views/play/CampaignView.coffee

```diff
@@ -94,10 +93,12 @@ module.exports = class CampaignView extends RootView
             if reward not in earned[group]
               console.warn 'Filling in a gap for reward', group, reward
               earned[group].push(reward)
-
     @supermodel.loadCollection(@earnedAchievements, 'achievements', {cache: false})
 
-    @listenToOnce @campaign, 'sync', @getLevelPlayCounts
+    @campaign = new Campaign({_id:@terrain})
+    @listenToOnce @campaign, 'sync', @onCampaignLoaded
+    @campaign = @supermodel.loadModel(@campaign).model
```

Comment:
The way supermodel works, the campaign you give it is not necessarily the one you get back, so the `@listenToOnce` should happen after `loadModel`.